Lib/idlelib/debugger.py

```diff
@@ -213,7 +213,7 @@ def interaction(self, message, frame, info=None):
                 m1 = "%s" % str(type)
             if value is not None:
                 try:
-                    m1 = "%s: %s" % (m1, str(value))
+                    m1 = f"{m1}: {str(value)}"
```

Comment:
str was redundant with %s.
```suggestion
                   # TODO redo entire section, trys not needed
                    m1 = f"{m1}: {str(value)}"
```